lib/features/thread/presentation/thread_controller.dart

```diff
@@ -377,6 +373,14 @@ class ThreadController extends BaseController with EmailActionController {
         await cachingManager.clearEmailCacheAndAllStateCache();
       }
       _getAllEmailAction();
+    } else if (error is MethodLevelErrors) {
+      if (currentOverlayContext != null && error.message?.isNotEmpty == true) {
+        appToast.showToastErrorMessage(
+          currentOverlayContext!,
+          error.message!
+        );
+      }
+      clearState();
```

Comment:
in what case we have this?

Reply:
When calling jmap request `Email/query` we get errors from jmap `MethodLevelErrors`. (During testing I encountered it several times on the preprod server). So it always shows the loading bar on the ThreadView screen so we should show the error message to the user and reset `ViewState` to hide the loading bar.